### ISSUE TITLE
server logs spurious "Invalid request range" error

### ISSUE THREAD

**Issue by krisis** [minio/minio#2165]
## Expected behaviour

server should not log error when http-range field in the header is empty.
## Actual behaviour

server logs "Invalid request range" error.
## Steps to reproduce the behaviour
- `mc cat alias/bucket/object | md5sum`
## Minio version
- master branch, commit id `bdff084`
## System information

`Linux trantor 4.4.0-28-generic #47-Ubuntu SMP Fri Jun 24 10:09:13 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by krisis** [comment]
Error message as seen in console,

```
ERRO[0005] Invalid request range                         cause='' does not start with 'bytes=' sysInfo=map[mem.total:285MB host.name:minio2 host.lang:go1.6 host.cpus:12 mem.used:83MB mem.heap.used:83MB mem.heap.total:270MB host.os:linux host.arch:amd64] type=*errors.errorString

```

**Comment by balamurugana** [comment]
Its already fixed by aa7079fc7b3ffef9f7f35fcc8167259f2c9594bc 

We can close this issue.
